nav2_map_server/test/component/test_map_saver_2d_node.cpp

```diff
@@ -83,7 +87,7 @@ class MapSaverTestFixture : public ::testing::Test
 protected:
   // Check that map_msg corresponds to reference pattern
   // Input: map_msg
-  void verifyMapMsg(const nav_msgs::msg::OccupancyGrid & map_msg)
+  static void verifyMapMsg(const nav_msgs::msg::OccupancyGrid & map_msg)
```

Comment:
What is this change about?

Reply:
A sugestion from Clang-Tidy https://clang.llvm.org/extra/clang-tidy/checks/readability-convert-member-functions-to-static.html

Reply:
While the function 
```c++ 
verifyMapMsg (const nav_msgs::msg::OccupancyGrid & map_msg)
```
is defined inside the class 
```c++
class MapSaverTestFixture{
  verifyMapMsg(...)
};
```
it isn't specific to class, and can be used independently even if no such object exist in there, that's why it was suggested that it is eligible to hold `static`.

Reply:
Maybe I am wrong, still analyzing the situation.

Reply:
Okay, I see no issue to make it static.